src/Acuminator/Acuminator.Analyzers/StaticAnalysis/DacExtensionDefaultAttribute/DacExtensionDefaultAttributeAnalyzer.cs

```diff
@@ -100,9 +101,14 @@ private static Task AnalyzePropertyAsync(SymbolAnalysisContext symbolContext, PX
 			}
 		}
 
-		private static  bool isAttributeContainsPersistingCheckNothing(KeyValuePair<string, TypedConstant> argument)
+		private static bool isAttributeContainsPersistingCheckNothing(KeyValuePair<string, TypedConstant> argument)
 		{
-			return (argument.Key.Contains(_PersistingCheck) && (int)argument.Value.Value == (int)PXPersistingCheck.Nothing);
+			return (
+				argument.Key.Contains(_PersistingCheck) &&
+					!argument.Value.IsNull && 
```

Comment:
The check `!argument.Value.IsNull` is not needed. The `argument.Value.Value is int intValue`  is enough.
Also the `intValue` or `boolValue` is not a good name for a variable - it is obvious from the code that it is int value. 
More semantically meaningful name related to the context should be used.